### DISCUSSION TITLE
mPDF grayscale

### DISCUSSION THREAD

**Discussion by nandi96** [mpdf/mpdf#1819] [Q&A]
Hi,

How can I modify the PHP to generate PDF only grayscale?

**Comment by finwe** [comment]
https://github.com/mpdf/mpdf-examples/blob/development/example56_new_mPDF_v5-1_features_grayscale.php

**Reply by finwe** [reply]
<strike>(But I vaguely remember that this example is now broken and should have been fixed).</strike>

**Reply by nandi96** [reply]
which file should I paste the code into?

**Reply by finwe** [reply]
Just add a `'restrictColorSpace' => 1` config option to your mPDF constructor.
